src/promptflow/promptflow/_sdk/_load_functions.py

```diff
@@ -62,19 +62,18 @@ def load_common(
 
 def load_flow(
     source: Union[str, PathLike, IO[AnyStr]],
-    connections=None,
-    variant=None,
-    **kwargs,
 ):
```

Comment:
add return annotation here?

Reply:
Shall we expose Flow entity?

Reply:
Offline synced, we'll keep it private for now and add in annotation.